api/src/services/user/user.hooks.ts

```diff
@@ -61,18 +65,29 @@ export default {
           'resetToken',
           'resetShortToken',
           'resetExpires',
+          'resetAttempts',
         ),
         commonHooks.iff(
           commonHooks.isNot(isUserProfile(USER_PROFILE.SUPERADMIN)),
-          commonHooks.preventChanges(true, 'email'),
-          commonHooks.preventChanges(true, 'blocked'),
+          commonHooks.preventChanges(true, 'email', 'blocked', 'password', 'profile'),
+          defineAbilitiesIffHook(),
+          authorize({
+            adapter: 'feathers-objection',
+          }),
         ),
         lowerCase('email'),
         hashPassword('password'),
         getCurrentItem(),
       ),
     ],
-    remove: [],
+    remove: [
+      commonHooks.iff(
+        (context: HookContext) => {
+          return !isUserProfile(USER_PROFILE.SUPERADMIN)(context) && commonHooks.isProvider('external')(context)
```

Comment:
Same here, a `ADMIN` could remove users.

Reply:
You've done it for the create / update, don't you think it's ok for the remove too ? (to add `ADMIN` ?)